docs/HOL_Continuous-Deployment-Using-Custom-Agent/README.md

```diff
@@ -0,0 +1,121 @@
+# HOL - Parts Unlimited MRP App Continuous Deployment with Visual Studio Team Services - Local Agent #
+
+In this lab, you will learn how to deploy the Parts Unlimited MRP App in an automated fashion onto a local agent on a Linux VM. After this lab, you will have a working, automated build in Visual Studio Team Services that will build, test, and deploy the Parts Unlimited MRP app to a Virtual Machine in Azure.
+
+>**Note:** If you would like to trigger continuous deployments using the hosted VSTS agent instead of a local agent, see [this lab](https://github.com/Microsoft/PartsUnlimitedMRP/tree/master/docs/HOL_Continuous-Deployment).  
+
+
+###Pre-requisites###
+
+- Completion of the Continuous Integration with Parts Unlimited MRP HOL
+- An active Azure Subscription
+- An active Visual Studio Team Services Account
+
+
+### Tasks Overview ###
+
+In this lab, you will work with one machine which will serve as both the deployment agent and the MRP server.
+
+**Provision the lab:** Provision a VSTS agent and MRP machine (Ubuntu VM) in Azure using an ARM template.
+
+**Configure the release definition:** Configure a release definition in VSTS that picks up build artifacts and triggers whenever new artifacts are produced. 
+
+**Trigger a build for continuous deployment:** Trigger a build for continuous deployment by making changes to code and automatically deploying the MRP app to the local agent.
+
+### Task 1: Provision the Lab ###
+
+1. Create the MRP agent pool in Visual Studio Team Services if you do not have one already. Go to the homepage of your VSTS account or the PartsUnlimitedMRP team project and clicking on the gear icon in the upper-right corner of the homepage.
```

Comment:
Go to the homepage of your VSTS account ~~or the PartsUnlimitedMRP team project~~